packages/core/admin/admin/src/constants.js

```diff
@@ -0,0 +1,92 @@
+        ],
+        create: [{ action: 'admin::roles.create', subject: null }],
+        delete: [{ action: 'admin::roles.delete', subject: null }],
+        read: [{ action: 'admin::roles.read', subject: null }],
+        update: [{ action: 'admin::roles.update', subject: null }],
+      },
+      users: {
+        main: [
+          { action: 'admin::users.create', subject: null },
+          { action: 'admin::users.read', subject: null },
+          { action: 'admin::users.update', subject: null },
+          { action: 'admin::users.delete', subject: null },
+        ],
+        create: [{ action: 'admin::users.create', subject: null }],
+        delete: [{ action: 'admin::users.delete', subject: null }],
+        read: [{ action: 'admin::users.read', subject: null }],
+        update: [{ action: 'admin::users.update', subject: null }],
+      },
+      webhooks: {
+        main: [
+          { action: 'admin::webhooks.create', subject: null },
+          { action: 'admin::webhooks.read', subject: null },
+          { action: 'admin::webhooks.update', subject: null },
+          { action: 'admin::webhooks.delete', subject: null },
+        ],
+        create: [{ action: 'admin::webhooks.create', subject: null }],
+        delete: [{ action: 'admin::webhooks.delete', subject: null }],
+        read: [
+          { action: 'admin::webhooks.read', subject: null },
+          // NOTE: We need to check with the API
+          { action: 'admin::webhooks.update', subject: null },
+          { action: 'admin::webhooks.delete', subject: null },
+        ],
+        update: [{ action: 'admin::webhooks.update', subject: null }],
+      },
+      'api-tokens': {
+        main: [{ action: 'admin::api-tokens.access', subject: null }],
+        create: [{ action: 'admin::api-tokens.create', subject: null }],
+        delete: [{ action: 'admin::api-tokens.delete', subject: null }],
+        read: [{ action: 'admin::api-tokens.read', subject: null }],
+        update: [{ action: 'admin::api-tokens.update', subject: null }],
+        regenerate: [{ action: 'admin::api-tokens.regenerate', subject: null }],
+      },
+      'transfer-tokens': {
+        main: [{ action: 'admin::transfer.tokens.access', subject: null }],
+        create: [{ action: 'admin::transfer.tokens.create', subject: null }],
+        delete: [{ action: 'admin::transfer.tokens.delete', subject: null }],
+        read: [{ action: 'admin::transfer.tokens.read', subject: null }],
+        update: [{ action: 'admin::transfer.tokens.update', subject: null }],
+        regenerate: [{ action: 'admin::transfer.tokens.regenerate', subject: null }],
+      },
+      'project-settings': {
+        read: [{ action: 'admin::project-settings.read', subject: null }],
+        update: [{ action: 'admin::project-settings.update', subject: null }],
+      },
+    },
+  };
+
+export const ADMIN_PERMISSIONS_SELECTOR = (state) => state.admin_app.permissions;
```

Comment:
Is a selector a constant or a helper/util? 😅 other selectors aren't capitalised is my only reservation – also do we expect to grow `admin_app`? If so should we use `createSelector` or is that overkill? 👀

Reply:
> 1. Is a selector a constant or a helper/util?

You are probably right and this should go into a file called `selectors.js`, to follow the other packages.

> 2. also do we expect to grow admin_app

Don't know. Probably. It was there before :)

> 3. If so should we use createSelector or is that overkill? 

Let me explore.

Good remarks!